CHANGELOG.md

```diff
@@ -72,6 +72,32 @@ SamplerDescriptor {
 ...
 }
 ```
+
+#### Multiview on all major platforms and support for multiview bitmasks
+
+Multiview has been reworked, adding support for DX12 and Metal, and adding testing and validation to wgpu itself.
```

Comment:
Should briefly clarify that this feature is _not_ just about multiple render targets but rather to have a single draw call get geometry instanced for each target including a way to dinstinguish said targets in shaders. Afaik the only mainstream usecase is AR/VR, so any explanation can always bring that up I figure.
It's imho a bit too niche to assume that everyone just knows what's meant with "multiview".